### DISCUSSION TITLE
Network initiated Device Status Subscription deletion

### DISCUSSION THREAD

**Discussion by vidhirikin27** [camaraproject/DeviceStatus#149] [General]
Hi all,

If the network itself deletes the subscription to the device status due to some reason, can the application that requested the subscription be notified?

Thanks,
Vidhi (Comviva)

**Comment by vidhirikin27** [comment]
@MarkusKuemmerle  Could you put me in touch with someone who can answer my question?

**Comment by vidhirikin27** [comment]
Awaiting response on this query. Anyone can guide on this?

**Comment by akoshunyadi** [comment]
@vidhirikin27 the application will receive subscription-ends event with the termination reason

**Reply by vidhirikin27** [reply]
Thanks @akoshunyadi so this should be informed by the network over the callback URL to the application?

**Reply by bigludo7** [reply]
@vidhirikin27 Yes ! 
This is described [here ](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#12-subscription-notification--event)
